lib/clover_field.cpp

```diff
@@ -373,6 +373,67 @@ namespace quda {
     checkCudaError();
   }
 
+  void cudaCloverField::prefetch(QudaFieldLocation mem_space, cudaStream_t stream) const
+  {
+    prefetch(mem_space, stream, CloverPrefetchType::BOTH_CLOVER_PREFETCH_TYPE);
```

Comment:
what about just naming the element `BOTH`. `Having CloverPrefetchType twice seems a bit wordy.